src/apns.app.src

```diff
@@ -6,6 +6,7 @@
   {applications, [
     kernel,
     stdlib,
+    jsx,
```

Comment:
We don't need to add jsx here. jsx is a library, not an application, so we don't have to start anything.